<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassWordGen</class>
 <widget class="QMainWindow" name="PassWordGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>447</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>447</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PassWordGen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>381</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="LengthLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Length</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lengthCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>150</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alphaNumCheckbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Alphanumeric Only</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>generateButton</sender>
   <signal>clicked()</signal>
   <receiver>PassWordGen</receiver>
   <slot>generate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PassWordGen</sender>
   <signal>passwordGenerated(QString)</signal>
   <receiver>passwordEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pasteButton</sender>
   <signal>clicked()</signal>
   <receiver>PassWordGen</receiver>
   <slot>copyclip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>passwordGenerated(QString)</signal>
  <slot>generate()</slot>
  <slot>copyclip()</slot>
 </slots>
</ui>
